CGPs/cgp-0140.md

```diff
@@ -3,7 +3,7 @@ cgp: 140
 title: Celo L2 off-chain data availability layer, powered by EigenLayer and EigenDA
 date-created: 2024-06-27
 author: cLabs Blockchain & Primitives teams
-status: DRAFT
+status: EXECUTED
 discussions-to: https://forum.celo.org/t/clabs-proposes-off-chain-data-availability-layer-powered-by-eigenlayer-and-eigenda/8236
 governance-proposal-id:
 date-executed:
```

Comment:
@ericnakagawa when you update the status to `"EXECUTED"` you will need to add `governance-proposal -id` and  `date-executed `values, otherwise the bot added by @jmrossy will show errors and Celo Mondo will go down too if PR approved.

Reply:
Ditto, see https://github.com/celo-org/governance/pull/462#discussion_r1707767983

Reply:
Cool @work-0xj4an can you make these changes to this PR. thanks.

Reply:
@ericnakagawa Don't know how. Closing this one and opening a new one.